python/whylogs/viz/utils/profile_viz_calculations.py

```diff
@@ -104,6 +104,47 @@ class DatasetSummary(TypedDict):
     properties: Optional[OverallStats]
 
 
+def generate_profile_summary(
+    target_view: DatasetProfileView, config: Optional[SummaryConfig]
+) -> Optional[Dict[str, Any]]:
+    if config is None:
+        config = SummaryConfig()
+
+    if not target_view:
```

Comment:
this behaviour should be tested, left a comment there on the test file

Reply:
added tests for it  in the last commit